WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1155,6 +1155,16 @@ FormAddRemoveButton          = @FormButtonHeight@
 FormButtonStyle              = @cssShadow@:0px 0px 0px transparent; padding:5px; @ActiveCursor@ @PanelBorder@ @MediumRoundCorners@ background-color:@WidgetBgColor@; @ButtonText@
 
 ###################
+#   SHORTCUT LIST
+###################
+PrefShortcutLabel            = min-width:50%; margin-bottom:4px; @FontSize-big@ @Text@
+PrefShortcutKey              = line-height:3.667em; margin:0 0.667em; padding:6px 10px; @Text@ @SeparatorBorder@ @MediumRoundCorners@
```

Comment:
Unitless value not adding any value as per my understanding here (as per the link's useful when we have child elements). Used relative sizing based on parent's font size.